build/charts/theia/provisioning/dashboards/networkpolicy_dashboard.json

```diff
@@ -1,647 +1,639 @@
 {
```

Comment:
It seems like the entire file has been changed due to some JSON re-formatting, which makes it extremely hard to tell which functionality code has been changed. 

Could you help verify only the following two changes mentioned by the PR description were made to this file?

3. Remove mean value in throughput related table
4. Add a threshold for time-series diagram to prevent points from connecting when the gap of time is > 90s

Reply:
Yes, there are 18 panels with these settings. So I removed 18 "means", "limit 50" and added 18 "spanNulls": 90000